src/commit.ts

```diff
@@ -196,7 +196,7 @@ export class ConventionalCommitMessage {
     this.description = metadata.description;
     this.footers = metadata.footers;
     this.scope = metadata.scope ? metadata.scope : null;
-    this.type = metadata.type ? metadata.type : null;
```

Comment:
`metadata.type` could not be `null`; removing incorrect annotation to reduce the number of suggested null-checks.